spec/lib/performance_platform/gateway/roaming_users_spec.rb

```diff
@@ -5,7 +5,8 @@
   let(:sessions) { DB[:sessions] }
   let(:ip_1) { "7.7.7.7" }
   let(:ip_2) { "8.8.8.8" }
-  let(:yesterday) { Date.today - 1 }
+  let(:today) { Date.today }
```

Comment:
Just `today`?

Reply:
This is to make sure that 'today' is a fixed date so that two statements that use 'today' are guaranteed to use the same value.